example/app.json

```diff
@@ -5,18 +5,13 @@
     "version": "0.0.1",
     "orientation": "portrait",
     "icon": "./assets/icon.png",
-    "userInterfaceStyle": "light",
     "sdkVersion": "51.0.0",
     "splash": {
       "image": "./assets/splash.png",
       "resizeMode": "cover",
       "backgroundColor": "#F8F9FF"
     },
     "ios": {
-      "entitlements": {
-        "com.apple.developer.applesignin": ["Default"]
-      },
-      "supportsTablet": true,
```

Comment:
I'd leave `supportsTablet` - we want to be able to test apple pencil, which only works on ipads.